src/components/scenes/GettingStartedScene.tsx

```diff
@@ -156,15 +162,44 @@ export const GettingStartedScene = (props: Props) => {
     }
   )
 
+  // Use state to track which button should be visible
+  const [showNextButton, setShowNextButton] = React.useState(false)
+
+  // Update the button visibility based on swipeOffset
+  useAnimatedReaction(
+    () => swipeOffset.value,
+    value => {
+      if (value > 0 && !showNextButton) {
+        runOnJS(setShowNextButton)(true)
+      } else if (value <= 0 && showNextButton) {
+        runOnJS(setShowNextButton)(false)
+      }
+    }
+  )
+
   const footerButtons = (
     <>
-      <ButtonsView
-        layout="column"
-        primary={{
-          label: lstrings.account_get_started,
-          onPress: handlePressSignUp
-        }}
-      />
+      <ButtonFadeContainer>
+        <EdgeAnim visible={!showNextButton} enter={fadeIn} exit={fadeOut}>
+          <ButtonsView
+            layout="column"
+            primary={{
+              label: lstrings.account_get_started,
+              onPress: handlePressGetStartedNext
+            }}
+          />
+        </EdgeAnim>
+
+        <EdgeAnim visible={showNextButton} enter={fadeIn} exit={fadeOut}>
+          <ButtonsView
+            layout="column"
+            primary={{
+              label: lstrings.string_next_capitalized,
+              onPress: handlePressGetStartedNext
+            }}
+          />
+        </EdgeAnim>
```

Comment:
I really wish we could just use a `Button` component and give it a `Text` component as a child so that way we can animate `Text`.

Reply:
Yeah this is a new use case that wasn't anticipated in the original design. A
good idea for future changes